autogpt_platform/backend/backend/data/credit.py

```
@@ -1776,7 +1776,9 @@ async def _schedule_downgrade_at_period_end(
 
 
 async def modify_stripe_subscription_for_tier(
-    user_id: str, tier: SubscriptionTier
+    user_id: str,
+    tier: SubscriptionTier,
+    billing_cycle: BillingCycle = "monthly",
```

Ruff (0.15.12)

[error] Undefined name `BillingCycle`

(F821)